README.md

```diff
@@ -45,68 +50,7 @@ const ReaderPage = ({ manifestUrl }) => {
 };
 ```
 
-Passing a content decryptor to the reader for use by the Client. This would be how we render AxisNow content for example:
-
-```typescript
-import WebReader from "nypl/web-reader"
-import AxisNowDecryptor from "nypl/axisnow-access-control-web"
-
-const ReaderPage = ({manifestUrl}) => {
-  const decryptor = new AxisNowDecryptor(...);
-  return (
-	  <WebReader
-      getContent={decryptor.getContent}
-      manifestUrl={manifestUrl}
-    />
-  )
-}
-```
-
-To support customization, you can piece together your own UI and call the `useWebReader` hook to get access to the reader API.
-
-```typescript
-import { useWebReader, ReaderNav, ReaderFooter } from 'nypl/web-reader';
-
-const CustomizedReaderPage = ({ webpubManifestUrl }) => {
-  // takes a manifest, instantiates a Navigator, and
-  // returns the Navigator, interaction handlers, and
-  // the current state of the reader as an object
-  const reader = useWebReader({
-    webpubManifestUrl,
-  });
-
-  return (
-    <div>
-      {/* eg. keep default header, but change its background */}
-      <ReaderNav {...reader} className="bg-blue" />
-
-      {/* we can add custom prev/next page buttons */}
-      <button onClick={reader.handleNextPage}>Next</button>
-      <button onClick={reader.handlePrevPage}>Prev</button>
-
-      {/* you will receive content from the reader to render wherever you want */}
-      {reader.content}
-
-      {/* use the default footer */}
-      <ReaderFooter {...reader} />
-    </div>
-  );
-};
-```
-
-If you know you are only going to be using one type of reader, you can also call the hook just for that reader:
-
-```typescript
-import { usePdfReader } from 'nypl/web-reader';
-
-const MyPdfReader = ({ webpubManifestUrl, manifest }) => {
-  const reader = usePdfReader({ manifest, webpubManifestUrl });
-
-  return <div>{reader.content}</div>;
-};
-```
-
-Finally, to use in a vanilla Javascript app:
+To use in a vanilla Javascript app:
 
 ```html
```

Comment:
Is this vanilla JS example helpful? I am not sure it is. If we packaged the web-reader with a regular CDN then this might be more useful, but since we don't I'm wondering if it makes sense to remove this example.

Reply:
Yea I think it's fine to remove this example... I don't remember exactly why we supported this in the first place